src/main/java/mate/academy/model/CartItem.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "cart_items")
+@SQLDelete(sql = "UPDATE cart_items SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted=false")
```

Comment:
Should it be soft-deleted?

Reply:
At first I did it without using soft-delete. But there were questions about this at the lectures, so we were told that it could be useful for collecting certain statistics (the task description requires soft-deletion, so there is an implementation)